htdocs/lib2/menu.class.php

```diff
@@ -34,7 +36,7 @@ public function __construct()
         // add country-dependent town list for small map
         $country = $login->getUserCountry();
         if ($opt['map']['towns']['enable'] &&
-            isset($build_map_towns_menu) && $build_map_towns_menu &&   // optimization
+            isset($build_map_towns_menu) && $build_map_towns_menu && // optimization
```

Comment:
I guess there are lots more of in-line comments then there _ever_ will be wrong spacings after logical operators. So I would prefer to turn this off and fix it manually.